Tools/LyTestTools/ly_test_tools/_internal/pytest_plugin/terminal_report.py

```diff
@@ -67,6 +69,17 @@ def pytest_terminal_summary(terminalreporter, exitstatus, config):
                 "(NOTE: The 'PYTHON' or 'PYTHONPATH' environment variables need values for accurate commands): ",
                 test_path, nodeids, build_dir)
 
+            # Check for unknown test failures
+            have_printed_header = False
```

Comment:
I don't see where this gets switched on if the header already exists

Reply:
Good catch